batch/test/test_batch.py

```diff
@@ -84,8 +84,8 @@ def test_msec_mcpu(self):
             job = self.client.get_job(job['batch_id'], job['job_id'])
             job = job.status()
 
-            # runs at 100mcpu
-            job_msec_mcpu2 = 100 * max(job['status']['end_time'] - job['status']['start_time'], 0)
+            # runs at 250mcpu (100mcpu rounds up to 250mcpu)
```

Comment:
100m was arbitrary, I don't think you need to keep that in the comment.

Reply:
I wanted to make sure the rounding up happened in an actual example, but got rid of the comment that it's rounded up.